### DISCUSSION TITLE
With GeoCompare On then OFF got "Internal Structure Out of Sync"

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#960] [General]
I did a short section of travel which I had done once previous.  I used GeoCompare to find the date of the prior travel with success.  I then turned OFF GeoCompare and then got an  Internal Structure Out of Sync popup.
Attached find screen shot of the Out of Sync and the mtb Error Log

![mtb Internal Structure Out of Sync](https://user-images.githubusercontent.com/106929478/200914022-d312e5ae-87fe-4b1a-b5c4-d00c16a83e1c.jpg)

[mtb Error Log 9Nov2022.txt](https://github.com/mytourbook/mytourbook/files/9974146/mtb.Error.Log.9Nov2022.txt)

**Comment by FJBDev** [comment]
@stvzldn 
is that something you experience every time ? Do you have a scenario that we could use to reproduce it locally in order to investigate the issue ?

**Comment by FJBDev** [comment]
And on a side note, because I've asked other users the same question, I see in your screenshot that you still import `.fitlogex` files. What do you use in ST that is not in MT  and that prevents you from importing activities directly to MT ?

**Comment by stvzldn** [comment]
The Out of Sync happens almost never and is not reproducible.  I think it is most likely to occur after a new tour is loaded and I examine that tour using GeoCompare.  Normally I have GeoCompare set OFF and only turn on if I want to check something (often to see my Strava segment history or when planning new Strava segments using mtb).  Is there a way to set Left and Right sliders using the 2D Tour Map instead of using the Tour Chart?  It could be that my struggle to set the L & R sliders also triggers a fault.

re: Side note - I use ST's equipment tracking extensively with some equipment distance based, some by use time, some by frequency.  I cannot seem to get the Tags to work well for equipment tracking.  I would like to be able to accumulate time/distance for tagged items.
I use the Training Load plugin with power based TSS, I use omb's plugin Course making function.  I use ST when comparing multiple activities.  I continue to accumulate ST activities as backup.
I had planned to only load to MTB back at year start, but instead just continued my usual workflow.  I also maintain a garmin training center dataset since I use GTC to load an activity tcx in Isaac for gps addition to my iBike Newton powermeter.  Too much duplicated data, but I have pretty much automated the workflow.

**Reply by wolfgang-ch** [reply]
> Is there a way to set Left and Right sliders using the 2D Tour Map instead of using the Tour Chart?

See https://github.com/mytourbook/mytourbook/issues/461
